Soccar [UI]/src/nl/soccar/ui/drawable/GameCanvas.java

```diff
@@ -27,8 +27,8 @@
      *
      * @param game The Game, not null, that will be used to interact with.
      */
-    public GameCanvas(Game game) {
-        engine = new GameEngine(game);
+    public GameCanvas(Session session) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): session [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)